doc/reference/reference_lua/log.rst

```diff
@@ -55,24 +31,61 @@ Below is a list of all ``log`` functions.
     +--------------------------------------+---------------------------------+
     | Name                                 | Use                             |
     +======================================+=================================+
+    | :ref:`log.cfg({})                    | Configures a logger             |
+    | <log-cfg>`                           |                                 |
+    +--------------------------------------+---------------------------------+
     | :ref:`log.error()                    |                                 |
     | <log-ug_message>` |br|               |                                 |
     | :ref:`log.warn()                     |                                 |
     | <log-ug_message>` |br|               |                                 |
-    | :ref:`log.info()                     | Write a user-generated message  |
-    | <log-ug_message>` |br|               | to a log file                   |
+    | :ref:`log.info()                     | Logs a message with the         |
+    | <log-ug_message>` |br|               | specified level                 |
     | :ref:`log.verbose()                  |                                 |
     | <log-ug_message>` |br|               |                                 |
     | :ref:`log.debug()                    |                                 |
     | <log-ug_message>`                    |                                 |
     +--------------------------------------+---------------------------------+
-    | :ref:`log.logger_pid()               | Get the PID of a logger         |
-    | <log-logger_pid>`                    |                                 |
+    | :ref:`log.pid()                      | Gets the PID of a logger        |
+    | <log-pid>`                           |                                 |
     +--------------------------------------+---------------------------------+
-    | :ref:`log.rotate()                   | Rotate a log file               |
+    | :ref:`log.rotate()                   | Rotates a log file              |
     | <log-rotate>`                        |                                 |
     +--------------------------------------+---------------------------------+
 
+.. _log-cfg:
+
+.. function:: log.cfg({})
+
+    Allows you to configure logging options.
```

Comment:
Maybe drop "Allows you to"?

Reply:
I'd keep it as is - I think, `Allows you to...` or `Provides the capability to ...` suits good for complex configuration options that don't do a specific thing but provide access to a bunch of related properties.